solidity/src/CadenceRandomConsumer.sol

```diff
@@ -56,6 +101,85 @@ abstract contract CadenceRandomConsumer is CadenceArchWrapper {
         return requestId;
     }
 
+    /**
+     * ----- REVEAL STEP -----
+     */
+
+    /**
+     * @dev This method fulfills a random request and returns a random number as a uint64.
+     *
+     * @param requestId The ID of the randomness request to fulfill.
+     * @return randomResult The random number.
+     */
+    function _fulfillRandomRequest(uint256 requestId) internal returns (uint64) {
+        bytes32 randomValue = _fulfillRandomness(requestId);
+        uint64 randomResult = uint64(uint256(keccak256(abi.encodePacked(randomValue, requestId))));
```

Comment:
This is where the hashing should become a PRG instantiation

Reply:
I've updated to include a PRG instance here